inst/rmarkdown/templates/springer/skeleton/skeleton.Rmd

```diff
@@ -1,63 +1,481 @@
 ---
-title: Title here
-subtitle: Do you have a subtitle? If so, write it here
-titlerunning: Short form of title (if too long for head)
-authorrunning: Short form of author list if too long for running head
-thanks: | 
-    Grants or other notes about the article that should go on the front 
-    page should be placed here. General acknowledgments should be placed at the
-    end of the article.
+# Supported options: 
+#   sn-nature:       Style for submissions to Nature Portfolio journals
+#   sn-basic:        Basic Springer Nature Reference Style/Chemistry Reference Style
+#   sn-mathphys:     Math and Physical Sciences Reference Style
+#   sn-aps:          American Physical Society (APS) Reference Style
+#   sn-vancouver:    Vancouver Reference Style
+#   sn-apa:          APA Reference Style 
+#   sn-chicago:      Chicago-based Humanities Reference Style
+#   default:         Default
+
+classoptions: 
+  - sn-basic      
+  - pdflatex      # Optional: Compile with pdflatex/xelatex
+  # - Numbered      # Optional: Use numbered references instead of namedate references (only for sn-nature, sn-basic, sn-vancouver, sn-chicago, sn-mathphys or sn-nature)
+  # - referee       # Optional: Use double line spacing 
+  # - lineno        # Optional: Add line numbers
+  # - iicol         # Optional: Double column layour
+
+# unnumbered: TRUE # For unnumbered level heads
```

Comment:
We have the `number_sections` in `pdf_document()` which is FALSE by default. 

I understand there is no conflict by unsetting this variable ? I would have expected that 
```
unnumbered: true
```
would be the default in that case... 

Maybe we should set `number_sections = TRUE` in the R function then ?

Or did you observe no effect at all ?